mobile/lib/features/trade/trade_bottom_sheet_tab.dart

```diff
@@ -46,6 +46,9 @@ class _TradeBottomSheetTabState extends State<TradeBottomSheetTab> {
 
   ChannelInfo? channelInfo;
 
+  /// The max channel capacity as received by the LSP or if there is an existing channel
```

Comment:
🤔 not sure if I follow this comment.

Does it mean that this is etheir:
- (if no channel open): potential channel capacity advertised by the LSP?
- (if channel open): current channel capacity (which could be different from the currently advertised by the LSP, right?)

Reply:
Yes, that's what I meant :)